code/modules/clothing/suits/ego_gear/he.dm

```diff
@@ -109,3 +109,13 @@
 							TEMPERANCE_ATTRIBUTE = 40
 							)
 
+/obj/item/clothing/suit/armor/ego_gear/solemnlament
+	name = "solemn lament"
+	desc = "The undertaker's outfit belongs to those who pay tribute to the dead.\
+	Only a solemn mind is required to express condolences; there is no need for showy accessories."
+	icon_state ="solemn_lament"
+	//The design philosophy was to be as faithful to the concept of Funeral of the Dead Butterflies as possible, but conversely be in line with the stat line with every other HE suit without having VAV tier resists.
+	armor = list (RED_DAMAGE = 0, WHITE_DAMAGE = 50, BLACK_DAMAGE = 50, PALE_DAMAGE = -50) //50
```

Comment:
Why does it have negative pale..?

Reply:
Partially balance, partially the fact that Funeral actively wants people to die.

Reply:
_also people just preferred this stat line over 0/2/5/0_

Reply:
If you want balance, change pale defenses to -25